src/tests/ftest/container/boundary.yaml

```diff
@@ -0,0 +1,37 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
```

Comment:
Does this test specifically need 4 server nodes? Could we achieve the same coverage by moving to HW Small and using 2 dual-engine server nodes?

Reply:
Dual engines per host might consume more testing time in large number of containers with io and potential stability issue.
We can add mux with dual engines if it is needed.

Reply:
Dual-engine isn't specifically needed. I just mean in terms of resources. Increased test time could be okay if it's not drastic. Since HW Small is half as many nodes as HW Medium, the overall resource cost is less. As for stability, is there a known issue? Shouldn't creating many pools/containers work with both single and dual engine? Or do you mean stability in test time?
If there are too many unknowns, it's okay to keep this as-is. But in general, we've been trying to reduce overall resource consumption of tests.

Reply:
I would keep as is,  
If we use dual servers per host, it still still to run on hw-medium.

Reply:
Just FYI: HW Small is 2 nodes. So 2 server nodes with dual-engine = 4 engines